add-on/TA-Demisto/README.md

```diff
@@ -3,12 +3,10 @@
 
 Supporting Add-on for Demisto. This application allows a user to create incident into Demisto from Splunk using custom alert action.
 
-
 # REQUIREMENTS
 
 * Splunk version 6.3 >=
-* This application should be installed on Search Head.
-
+* This application should be installed on Search Head or Universal forwarder, Indexer, and Search Head if applicable.
```

Comment:
Why search head twice?

Reply:
@orius123 it's two different installation constellation. Thought it's better to specify instead of saving in words